server/src/main/java/com/thoughtworks/go/server/service/TimerScheduler.java

```diff
@@ -54,32 +52,25 @@
 
     private GoConfigService goConfigService;
     private BuildCauseProducerService buildCauseProducerService;
-    private SchedulerFactory quartzSchedulerFactory;
     private ServerHealthService serverHealthService;
     private Scheduler quartzScheduler;
-    protected static final String QUARTZ_GROUP = "CruiseTimers";
+    protected static final String PIPELINE_TRIGGGER_TIMER_GROUP = "PIPELINE_TRIGGGER_TIMER_GROUP";
     protected static final String BUILD_CAUSE_PRODUCER_SERVICE = "BuildCauseProducerService";
     protected static final String PIPELINE_CONFIG = "PipelineConfig";
 
     @Autowired
-    public TimerScheduler(SchedulerFactory quartzSchedulerFactory, GoConfigService goConfigService,
+    public TimerScheduler(Scheduler scheduler, GoConfigService goConfigService,
                           BuildCauseProducerService buildCauseProducerService, ServerHealthService serverHealthService) {
         this.goConfigService = goConfigService;
         this.buildCauseProducerService = buildCauseProducerService;
-        this.quartzSchedulerFactory = quartzSchedulerFactory;
+        this.quartzScheduler = scheduler;
         this.serverHealthService = serverHealthService;
     }
 
     public void initialize() {
-        try {
-            quartzScheduler = quartzSchedulerFactory.getScheduler();
-            quartzScheduler.start();
-            scheduleAllJobs(goConfigService.getAllPipelineConfigs());
-            goConfigService.register(this);
-            goConfigService.register(pipelineConfigChangedListener());
-        } catch (SchedulerException e) {
-            showGlobalError("Failed to initialize timer", e);
-        }
+        scheduleAllJobs(goConfigService.getAllPipelineConfigs());
+        goConfigService.register(this);
+        goConfigService.register(pipelineConfigChangedListener());
```

Comment:
Just moved this initialization in the spring configuration, so what we inject now is a scheduler, and not a factory.